packages/devtools-launchpad/src/index.js

```diff
@@ -144,6 +143,12 @@ async function getTabs(actions) {
   actions.newTabs(firefoxTabs);
 }
 
+async function getTabs(actions) {
+  actions.clearTabs();
```

Comment:
This won't pickup newly started firefox instances. If you do:
- yarn start in launchpad
- go to localhost:8000
- click on "Launch Firefox"
=> you now have to reload the page to see the tab from the new firefox instance
(previously it was automatically picked up)

If the connection was not established, maybe we should still try to do connectClients()?